var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -79,6 +81,7 @@ class Phist(CMakePackage):
     depends_on('cmake@3.8:', type='build')
     depends_on('blas')
     depends_on('lapack')
+    depends_on('python@3')
```

Comment:
Is it not version specific? Say starting with 1.6.1? If so, you should add ‘when=‘